src/gui/styles/types/custom_styles/dracula.rs

```diff
@@ -0,0 +1,28 @@
+#![allow(clippy::unreadable_literal)]
+
+//! Dracula theme
+//! <https://draculatheme.com/>
+use iced::color;
+
+use super::{CustomPalette, Palette, PaletteExtension};
+
+pub(super) fn dracula() -> CustomPalette {
+    CustomPalette {
+        name: "Dracula",
+        palette: Palette {
+            primary: color!(0x44475a),          // Current line
+            secondary: color!(0xff79c6),        // Pink
+            outgoing: color!(0x8be9fd),         // Cyan
+            buttons: color!(0x6272a4),          // Comments
+            text_headers: color!(0x44475a),     // Current line
+            text_body: color!(0xf8f8f2),        // Foreground
+            round_borders: color!(0xbd93f9),    // Purple
```

Comment:
for what concerns round_borders and round_containers I had an idea.
As you can see, I always used the same color as buttons for both, just with a different opacity.
This makes the UI feel less heavy and more harmonious in my opinion.
So: buttons_opacity is 1.0, round_borders opacity is less than 1.0 and greater than round_containers opacity, which in turn is greater than 0.0.
If you agree, I'll refactor palettes to only include the opacity values and the color are not needed anymore since they'll use the same as the buttons

Reply:
Sounds good. I originally didn't know what to set for those colors because they tended to be the same or similar to the buttons color.

You can experiment with it with this patch to see if the change is aesthetically pleasing with the original themes as well as the extras. :smiley_cat: